Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputController.cs

```diff
@@ -148,20 +148,156 @@ internal enum DeviceType
         /// Controller subtype enumeration in <c>SubType</c> field of <c>XINPUT_CAPABILITIES</c>.
         /// </summary>
         /// <remarks>
-        /// See <a href="https://docs.microsoft.com/en-us/windows/win32/xinput/xinput-and-controller-subtypes">MSDN</a>.
+        /// Provides additional detail about the underlying hardware being used and how it maps physical to logical
+        /// controls.
+        ///
+        /// See <a href="https://docs.microsoft.com/en-us/windows/win32/xinput/xinput-and-controller-subtypes">MSDN</a>
+        /// for additional details.
         /// </remarks>
         public enum DeviceSubType
         {
+            /// <summary>
+            /// The controller type is unknown.
+            /// </summary>
             Unknown = 0x00,
+
+            /// <summary>
+            /// Gamepad controller.
+            /// </summary>
+            /// <remarks>
+            /// Includes left and right stick as <see cref="Gamepad.leftStick" /> and <see cref="Gamepad.rightStick"/>,
+            /// left and right trigger as <see cref="Gamepad.leftTrigger"/> and <see cref="Gamepad.rightTrigger"/>,
+            /// directional pad as <see cref="Gamepad.dpad"/>,
+            /// and all standard buttons (<see cref="Gamepad.buttonSouth"/>, <see cref="Gamepad.buttonEast"/>,
```

Comment:
Slightly odd order SEWN
I would expect NESW (i.e clockwise) or alphabetical but its not very important ;)

Reply:
I kept MSDNs order

Reply:
All of these are basically straight off transformations of MSDNs definitions

Reply:
SEWN is a proper order, it's debatable whether SWEN is also valid based on the HID standards definition on how to order buttons, e.g. S is button0 since it has the minimum distance from the resting position of the finger used to press it, i.e. its the most ergonomically fit button (primary).